web/client/translations/data.it-IT

```diff
@@ -492,7 +492,7 @@
         "shapefile": {
             "title": "Carica Shapefile",
             "tooltip": "Aggiungi uno shapefile alla mappa.",
-            "placeholder": "Trascina qui il file o fai click per selezionare lo shapefile da importare.",
+            "placeholder": "Trascina qui il file o fai click per selezionare lo shapefile da importare.(gli shapefiles devono essere contenuti in archivi zip)",
```

Comment:
sorry, my fault. The complete string should be:
`Trascina qui i file o fai click per selezionare gli shapefile da importare.(gli shapefiles devono essere contenuti in archivi zip)